kommunicateui/src/main/java/com/applozic/mobicomkit/uiwidgets/conversation/activity/ConversationActivity.java

```diff
@@ -554,6 +582,23 @@ protected void onActivityResult(int requestCode, int resultCode, Intent data) {
                 }
                 return;
             }
+
+            if (requestCode == REQUEST_CODE_MULTI_SELECT_GALLERY) {
+                try {
+                    final ClipData clipData = data.getClipData();
+                    if (clipData == null) {
+                        final Uri singleFileUri = (data == null ? null : data.getData());
+                        kmAttachmentsController.processFile(singleFileUri, alCustomizationSettings, prePostUIMethods);
+                    } else {
+                        final int dataCount = clipData.getItemCount();
+                        for (int i = 0; i < dataCount; i++) {
```

Comment:
@shubhamtewari is dataCount variable used anywhere? if not, then we can directly write i < clipData.getItemCount()

Reply:
done